tasks/actorstate/miner/sector_events.go

```diff
@@ -161,117 +246,136 @@ func ExtractMinerPreCommitEvents(extState extraction.State, preCommits *miner.Pr
 	return out
 }
 
-func ExtractMinerPartitionsDiff(ctx context.Context, extState extraction.State, dlDiff miner.DeadlinesDiff) (minermodel.MinerSectorEventList, error) {
-	_, span := otel.Tracer("").Start(ctx, "ExtractMinerPartitionsDiff") // nolint: ineffassign,staticcheck
-	defer span.End()
-
-	// short circuit genesis state.
-	if extState.ParentState() == nil {
-		return nil, nil
-	}
-
-	if dlDiff == nil {
-		return nil, nil
-	}
-
-	removed := bitfield.New()
-	faulted := bitfield.New()
-	recovered := bitfield.New()
-	recovering := bitfield.New()
+// SectorStates contains a set of bitfields for active, live, fault, and recovering sectors.
```

Comment:
Would be good to mention how this bitfield represents each type of sector represented here. (a packed binary list of sector IDs?)